packages/flutter/lib/src/widgets/focus_manager.dart

```diff
@@ -759,6 +759,16 @@ class FocusNode with DiagnosticableTreeMixin, ChangeNotifier {
     return null;
   }
 
+  /// Returns the amount of additional space to reveal around the attached widget
+  /// when focused inside a scrolling container.
+  ///
+  /// For example, a value of `EdgeInsets.all(16.0)` ensures 16 pixels of
+  /// the adjacent widget are visible when this node receives focus.
+  ///
+  /// By default, this returns [FocusManager.defaultScrollRevealPadding] from the
+  /// associated [FocusManager], or [EdgeInsets.zero].
+  EdgeInsets get scrollRevealPadding => _manager?.defaultScrollRevealPadding ?? EdgeInsets.zero;
```

Comment:
I think keeping the naming conventions related makes it clearer where and how this will be used.

Reply:
Also, this should probably reference the ensureVisible function.

Reply:
The name may seem clunky to some, but it has the benefit of unmistakable relation to the associated function call.

Reply:
I've updated the documentation to reference `Scrollable.ensureVisible`. Should I update the name still?

Reply:
Please and thanks! It makes it easier to correlate the two with each other.